sklearn/_config.py

```diff
@@ -249,6 +261,13 @@ def config_context(
 
         .. versionadded:: 1.2
 
+    skip_parameter_validation : bool, default=None
+        If True, disable the validation of the hyper-parameters of estimators and
+        arguments passed to public functions. It can save time in some situations but
+        can lead to potential crashes.
```

Comment:
```suggestion
        If True, disable the validation of the hyper-parameters values in the fit
        method of estimators and for arguments passed to public functions. It can
        save time in some situations but can lead to low level crashes and
        exceptions with confusing error messages.
```